pkgs/applications/networking/sync/rclone/deps.nix

```diff
@@ -1,219 +0,0 @@
-# This file was generated by go2nix.
```

Comment:
As far as I can tell, this file is no longer necessary. The rclone project pins and includes its own go dependencies under the `vendor` subdirectory. See https://github.com/ncw/rclone/tree/master/vendor
Moreover, go2nix has been updated to take the `vendor` sudirectory into account when computing deps.nix. See [go2nix#21](https://github.com/kamilchm/go2nix/issues/21) In fact, running go2nix on this project now produces an empty deps.nix file.